wbooks/src/app/components/TabNavigator/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import libraryIcon from '@assets/ic_library.png';
+import libraryIconActive from '@assets/ic_library_active.png';
+import wishlistIcon from '@assets/ic_wishlist.png';
+import wishlistIconActive from '@assets/ic_wishlist_active.png';
+import TabIcon from '@components/TabIcon';
+import colors from '@constants/colors';
+import SCREENS from '@constants/screens';
+import { bottomTabParamList } from '@interfaces/navigatorParamLists';
+import EmptyView from '@screens/EmptyView';
+import Library from '@screens/Library';
+
+const Tab = createBottomTabNavigator<bottomTabParamList>();
+
+const TabNavigator = () => {
+  return (
+    <Tab.Navigator tabBarOptions={{ activeTintColor: colors.cerulean, inactiveTintColor: colors.dustyGray }}>
+      <Tab.Screen
+        name={SCREENS.LIBRARY}
+        component={Library}
```

Comment:
Here must be the `Library` stack, not only the screen

Reply:
Created a new stack navigator to display the Library as its only route